lib/gc.js

```diff
@@ -64,11 +64,14 @@ class GCEntry {
   [kSample] (ns) {
     this[kTotalDuration] += ns
     /**
-     * We have to truncate the value here because `record`
+     * We have to adjust the value here because `record`
      * only accepts integer values:
      * https://github.com/nodejs/node/blob/cdad3d8fe5f468aec6549fd59db73a3bfe063e3c/lib/internal/histogram.js#L283-L284
      */
-    this[kHistogram].record(Math.trunc(ns))
+    const val = Math.floor(ns)
```

Comment:
@andrewpuch Here we don't truncate anymore, but didn't see any difference, do you see any issues with it?

Reply:
Sorry, there's `round` not `floor` I have forced pushed.